pkgs/development/lua-modules/overrides.nix

```diff
@@ -55,4 +55,17 @@ with super;
       install -D completions/zsh/_busted $out/share/zsh/site-functions/_busted
     '';
   });
+
+  luuid = super.luuid.override({
+    buildInputs = [ pkgs.libuuid ];
+    extraConfig = ''
+      variables = {
+        LIBUUID_INCDIR="${pkgs.lib.getDev pkgs.libuuid}/include";
+        LIBUUID_LIBDIR="${pkgs.lib.getLib pkgs.libuuid}/lib";
```

Comment:
luarocks seems a bit broken because it did not add `LIBUUID_INCDIR` correctly to `CFLAGS` some how and I had to add it to buildInputs as well.